daemon/health.go

```diff
@@ -25,6 +25,9 @@ const (
 	// Also the time before the first probe.
 	defaultProbeInterval = 30 * time.Second
 
+	// Default interval between the container starting and the first probe run.
```

Comment:
The comments seems not correct.  `Start interval is the time between health checks during the start period` https://docs.docker.com/reference/dockerfile/#healthcheck

Reply:
I rebased and fixed the description